package.json

```diff
@@ -50,6 +50,7 @@
     "@ensdomains/ens-022": "npm:@ensdomains/ens@0.2.2",
     "@ensdomains/ethregistrar": "^1.2.2",
     "@ensdomains/resolver": "^0.1.6",
+    "@ethvault/iframe-provider": "^0.1.8",
```

Comment:
Is there anyway we can  pin it to a hash first or point to the specific commit/version of github to prevent accidentally upgrading to a version which could have malicious version when doing `npm update "@ethvault/iframe-provider"` ? That's requested by @Arachnid

Reply:
I added a package.json.test.js file so it is not possible to accidentally upgrade the iframe provider version

Can add other things to this test like checking that no versions are not specific, i.e. do not contain ^

Reply:
Let's just lock the version to 0.1.8, rather than ^0.1.8.

Reply:
Done

Reply:
Locked to 0.1.9, 0.1.9 changes the JSON RPC request IDs to numbers because older web3 versions check this
https://github.com/ethvault/iframe-provider/commit/d22a4c7543152e92ec41e97b874d4c6b93e26d6d